### ISSUE TITLE
AttributeError: 'dict' object has no attribute 'insert'

### ISSUE THREAD

**Issue by tomsommer** [pannal/Sub-Zero.bundle#280]
```
2017-05-15 16:24:54,614 (7f5dc0d57700) :  CRITICAL (agentkit:1078) - Exception in the update function of agent named 'Sub-Zero Subtitles (TV, 1.4.27.980)', called with guid 'com.plexapp.agents.thetvdb://262980/1/1?lang=en' (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-995f1dead/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/agentkit.py", line 1076, in _update
    agent.update(obj, media, lang, **kwargs)
  File "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins/Sub-Zero.bundle/Contents/Code/__init__.py", line 228, in update
    subtitle=subtitle)
  File "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Plug-ins/Sub-Zero.bundle/Contents/Libraries/Shared/subzero/history_storage.py", line 81, in add
    items.insert(0, item)
AttributeError: 'dict' object has no attribute 'insert'


```

**Comment by pannal** [comment]
Which SZ version are you using?

**Comment by tomsommer** [comment]
It says in the error?

**Comment by pannal** [comment]
Right, sorry :) 

That'll be fixed in 2.0, which is approaching final steps.
